pkg/resources/environments/environment.go

```diff
@@ -96,7 +96,7 @@ func DetectOpenshift() (bool, error) {
 	return err == nil, nil
 }
 
-func AddEnvVarForBasic(requireLogin string, journalType string) []corev1.EnvVar {
+func AddEnvVarForBasic(requireLogin string, journalType string, jolokiaAgentEnabled string, managementRBACEnabled string) []corev1.EnvVar {
```

Comment:
@brusdev a part of the idea here with keeping the original versions was to keep the original code intact and untouched leaving the previous implementation as is in case needed. That would also however mean new func's would need to be added which built on the previous one instead of being changed directly.

Reply:
I have just reverted this change and created new environment functions.